### DISCUSSION TITLE
Resolving AdGuards Hostname itself cause conflict using TLS certificate

### DISCUSSION THREAD

**Discussion by whisky23** [AdguardTeam/AdGuardHome#4660] [Q&A] [answered]
Hi,
I'd like to harden my AdGuard Home using the GUI over HTTPS.
As TLS certificates requires a hostname as CN/DNS-Name and not an IP, I'd like to access my Web-GUI via Hostname like "https://myadguard/"
This works fine for other local hostnames by setting several filters like
/^mydockerhost/$dnsrewrite=192.168.1.35
(I'm using this, because it's more flexible than DNS Rewrite)

Solely the rule for AdGuard itself fails:
/^myadguard/$dnsrewrite=192.168.1.99
this leads to 

```
dig myadguard @192.168.1.99
...
;; ANSWER SECTION:
myadguard.		10	IN	A	127.0.1.1
```

Has anybody an idea, why this fails or how I can use my AdGuard Home using the Hostname and not the IP?

**Comment by ainar-g** [comment, marked as answer]
Hello.  Look in the Query log.  It's probably because you also have `myadguard` in your `/etc/hosts` file.  You can either delete it (or comment it out) or disable `/etc/hosts` resolving completely through the configuration file.

**Comment by whisky23** [comment]
Hi ainae-g, a very good tip!
I forgot that this file is managed by my hypervisor, so after defining an exception for this file, now everything is fine!
Is there a special option to disable it by adguard configuration file? I've found the CLI-option   --no-etc-hosts, but it is marked as deprecated.
I've solved it by commenting out in the /etc/hosts directly for now. 
Thank you very much for your help!

**Reply by ainar-g** [reply]
@whisky23, see the *Configuration Changes* section in the [v0.107.7 release notes](https://github.com/AdguardTeam/AdGuardHome/releases/tag/v0.107.7).
